src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.abstractions.Figure;
+import core.basesyntax.enums.Figures;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Circle CIRCLE = new Circle("white", 10);
+    private static final double BOUND = 200;
+
+    private double getRandomNumber() {
+        return new Random().nextDouble(BOUND);
+    }
+
+    public Figure getRandomFigure() {
+        ColorSupplier color = new ColorSupplier();
+
+        int index = new Random().nextInt(Figures.values().length);
```

Comment:
random should be private final class-level field
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level